docs/content/preview/yugabyte-voyager/known-issues/postgresql.md

```diff
@@ -1615,3 +1605,190 @@ Suggested change to the schema is to remove this redundant index `idx_orders_ord
+
+#### Multi-column index
+
+In a multi-column index with a low cardinality column as the sharding key, the data will be unevenly distributed.
+
+**Workaround**:
+
+Make this index a range-sharded index to distribute data based on the combined values of all columns, or reorder the index columns to place the high-cardinality column first. This enables sharding on the high-cardinality column and ensures even distribution across all nodes.
+
+**Example**:
+
+An example schema on the source database is as follows:
+
+```sql
+CREATE TYPE order_statuses AS ENUM ('CONFIRMED', 'SHIPPED', 'OUT FOR DELIVERY', 'DELIVERED', 'CANCELLED');
+
+CREATE TABLE orders (
+    order_id int PRIMARY,
+    ...,
+    status order_statuses
+);
+
+CREATE INDEX idx_order_status on orders (status); --single column index on column having only 5 values 
+
+CREATE INDEX idx_order_status_order_id on orders (status, order_id); --multi column index on first column with only 5 values 
+```
+
+Since the no. of distinct values of the column status is 5, there will be a maximum of 5 tablets created, limiting the scalability.
+
+Suggested change to both types of indexes
+
+To make it a multi-column range-index:
+
+```sql
+ --These indexes will distribute the data on the combine value of both and as order_id is high cardinality column, it will make sure that data is distributed evenly
+
+CREATE INDEX idx_order_status on orders(status ASC, order_id); --adding order_id and making it a range-sharded index explictly
+
+CREATE INDEX idx_order_status_order_id on orders (status ASC, order_id); --making it a range-sharded index explictly 
+```
+
+OR
+
+multi-column with a sharding key on a high-cardinality column:
+
+```sql
+--these indexes will distribute the data on order_id first and then each shard is clustered on status  
+
+CREATE INDEX idx_orders_status on orders(order_id, status); --making it multi column by adding order_id as first column 
+
+CREATE INDEX idx_order_status_order_id on orders (order_id, status); --reordering the columns to place the order_id first and then keeping status.
+```
+
+---
+
+### Index on column with a high percentage of NULL values
+
+**Description**:
+
+In YugabyteDB, the columns specified in the [CREATE INDEX](../../../api/ysql/the-sql-language/statements/ddl_create_index) statement are of three kinds: sharding, clustering, and covering. For more details, refer to the documentation for [secondary indexes](../../../explore/ysql-language-features/indexes-constraints/secondary-indexes-ysql/). By default, the sharding strategy is HASH unless [Postgres Compatibility mode](./../../develop/postgresql-compatibility/) is enabled, in which case RANGE is the default strategy.
+
+The index must be designed in a way that the data is evenly distributed among all the nodes and is performant as per the query patterns.
```

Comment:
we can keep the wording similar for these two issues as the low-cardinality one above.

> The sharding strategy is HASH by default unless [Postgres Compatibility mode](./../../develop/postgresql-compatibility/) is enabled, in which case, RANGE is the default sharding strategy.

> Design the index to evenly distribute data across all nodes and optimize performance based on query patterns.